kytos/core/kytosd.py

```diff
@@ -37,25 +37,24 @@ def _create_pid_dir():
 
 def start_shell(controller=None):
     """Load Kytos interactive shell."""
+    documentation = "https://github.com/kytos-ng/" \
+                    "documentation/tree/master/tutorials/napps"
     kytos_ascii = r"""
-      _          _
-     | |        | |
-     | | ___   _| |_ ___  ___
-     | |/ / | | | __/ _ \/ __|
-     |   <| |_| | || (_) \__ \
-     |_|\_\__,  |\__\___/|___/
-            __/ |
-           |___/
+     _   __      _
+    | | / /     | |
+    | |/ / _   _| |_ ___  ___ ______ _ __   __ _
+    |    \| | | | __/ _ \/ __|______| '_ \ / _` |
+    | |\  \ |_| | || (_) \__ \      | | | | (_| |
```

Comment:
![20221121_135200](https://user-images.githubusercontent.com/1010796/203113404-1d11a1d5-0e9a-455a-acdd-931f99353973.png)

@Alopalao, could you try to set the hyphen to be centered vertically? 

I have an impression that as it is with this font it ended up too high.

Reply:
Sure